bit_manipulation/odd_or_even.py

```diff
@@ -0,0 +1,17 @@
+def odd_or_even(n: int) -> str:
+    """
+    whether a given number is odd or even using bitwise AND
+    >>> odd_or_even(1)
+    Odd
+    >>> sum_of_digits(14)
```

Comment:
What's `sum_of_digits`?

Reply:
Copy/paste error from https://github.com/TheAlgorithms/Python/blob/master/maths/sum_of_digits.py

Reply:
Now I have changed the code and documentations